packages/embed/src/utils/create-iframe/get-form-event-handler.ts

```diff
@@ -1,18 +1,18 @@
-import { ActionableOptions } from '../../base'
+type cb = (ev?: any) => void
```

Comment:
Please name this properly.

Reply:
any suggestion since I cannot use `callback` or `function`

Reply:
using `callbackFn` by now